src/instructlab/utils.py

```diff
@@ -367,7 +367,9 @@ def get_sysprompt(model=None):
     Returns:
         str: The system prompt for the model being used
     """
-    return common.SYS_PROMPT
+    config = read_config()
```

Comment:
It looks like you can get rid of the `model` argument here too, As of this change it doesn't look like we have plans to implement it.

Reply:
hey thanks for your feedback. Seems I see @jaideepr97 working on another pr that related to this prompt? Should I continue on this?

Reply:
Is this the PR you're mentioning: https://github.com/instructlab/instructlab/pull/2592? 

From what I can tell, it looks like you're trying to give the user the ability to set the system prompt if they wish, which I think is great, and I don't see that functionality in #2582.

If that's the case, see how he feels about the changes here, and possibly expect to rebase and merge this PR in after he's done with his.

Reply:
> It looks like you can get rid of the `model` argument here too, As of this change it doesn't look like we have plans to implement it.

yeah, model put it in there for a long time